### PR TITLE
Load all video entries

### PR DESCRIPTION
**Pull request by pinoelefante**
Addon now fetch all entries

Fix issue #42

### CODE REVIEW

**Comment by trummerjo** [comment]
does this PR affect the loading times of the lists?

**Comment by hawkeyexp** [comment]
tried and seems to work good on my systems

**Comment by pinoelefante** [comment]
@trummerjo being multiple calls to netflix api, it does. but not so much.
this addon load categories only from the homepage and this lists are composed of about 40 entries, so there is only one extra call. so there is not so much extra time

**Comment by asciidisco** [comment]
Some of the lists that could be requested may have up to thousands of possible video entries, if I request the "Documentaries" list (in my profile), I get 2032 video entries back (not tested with your PR, I just queried the API). I might be mistaken, as I just had the time to briefly have a look at this here. is this addressed somehow?

Regardless of the questions @trummerjo & I have, I wanted to say: Thank you for tackling this 👍

**Comment by pinoelefante** [comment]
@asciidisco I got only categories from the homepage. I can't test with full categories. The larger list I can test is "My list". It has 120 entries and it takes about 5 seconds to load.
With hundreds entries it will take a tons of time. This could be a temporary fix until a proper fix.

**Comment by trummerjo** [comment]
The best "fix" in my opinion would be if kodi fires an event when the bottom of a list is reached so that the plugin could load more data. I'm not a fan of the "traditional" pagination with folders etc.
But that would require a lot of changes in kodi i think.

**Comment by bhaal2** [comment]
If just selecting a list such as "Documentaries" are there not subgenres under that?  Could this not be used first to narrow the list?  I'm in Australia, so we only get a small amount of the content that NF has for most other countries etc..

**Comment by asciidisco** [comment]
@pinoelefante Thank you for the additions. Pagination isn't my favourite way to do it, but I´d say it´s better to have this, instead of not being able to access the content at all.

@trummerjo What´s your take on this? Are you okay with merging it until better options are available?

**Comment by trummerjo** [comment]
Yes would be better to have something - so im ok with it.

File: resources/lib/KodiHelper.py [hunk 1]
```diff
@@ -473,11 +473,17 @@ def build_video_listing (self, video_list, actions, type, build_url):
                 url = build_url(params)
             xbmcplugin.addDirectoryItem(handle=self.plugin_handle, url=url, listitem=li, isFolder=isFolder)
 
+        if has_more:
+            li_more = xbmcgui.ListItem(label="[COLOR cyan][B]-- NEXT PAGE --[/B][/COLOR]")
```
Comment (trummerjo) [inline resources/lib/KodiHelper.py]: Please make this one translatable

**Comment by pinoelefante** [comment]
@trummerjo done

File: resources/language/English/strings.po [hunk 2]
```diff
@@ -164,3 +164,7 @@ msgstr ""
 msgctxt "#30036"
 msgid "Always use the original title on export"
 msgstr ""
+
+msgctxt "#30045"
```
Comment (hawkeyexp) [inline resources/language/English/strings.po]: Can you change to 30046 cause it conflicts with my commits for views.

Comment (hawkeyexp) [reply resources/language/English/strings.po]: Forget it was only a line break sorry!

**Pull request merged**